framework/ui/src/index.ts

```diff
@@ -63,7 +63,11 @@ export default {
         return true;
       }
       // Check if email
-      if (!/^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/.test(value)) {
+      if (
+        !/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)+$/.test(
```

Comment:
Do we need such a huge rule for the email? Does it reflects the platform rule for emails?